Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -163,6 +164,45 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
           "res/actions/window.png")
       .SetHelpPath("/interface/scene-editor/events")
       .AddCodeOnlyParameter("currentScene", "");
+
+  extension
+      .AddAction("PrioritizeLoadingOfLayout",
+                 _("Preload scene"),
+                 _("Preload a scene assets as soon as possible in background."),
+                 _("Preload scene _PARAM1_ in background"),
+                 "",
```

Comment:
There is only one action, it would look odd, unless if it's for hiding it, but it would make an extra click for users without saving any space in the list.

Reply:
Maybe we could explain in the description that most games don't need to worry about it?

Reply:
Yes please. Use a Hint in newIDE/app/src/Hints/index.js to do so. Thanks!
"For most games, the default, automatic loading of resources will be sufficient. You can use this action to ensure you start loading a specific scene. This is helpful if you want to use the expressions related to loading to display a progress/loading bar."